src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final FigureName figureName = FigureName.ISOSCELES_TRAPEZOID;
+    private static final int DIVIDER = 2;
+    private double height;
+    private double firstSide;
+    private double secondSide;
+
+    public IsoscelesTrapezoid(String color, double height, double firstSide, double secondSide) {
+        super(color);
+        this.height = height;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide + secondSide) * height / DIVIDER;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure:\t" + figureName
```

Comment:
Use name() for getting String representation of enum constants. Replace figureName with figureName.name().